AffiliatedPackages.md

```diff
@@ -0,0 +1,111 @@
+# PlanetaryPy Affiliated Packages
+
+PlanetaryPy affiliated packages are an important aspect of the 
+PlanetaryPy community.
+
+An affiliated package is a planetary science related Python package that
+is not part of the `planetarypy` core package, and is not managed by the
+project but is a part of the PlanetaryPy Project community. These
+packages demonstrate a commitment to PlanetaryPy’s goals of improving
+reuse, interoperability, and interface standards for Python planetary
+science packages. In many (but not all) cases, affiliated
+packages also follow similar devlopment processes and package
+templates as for the core package.
+
+PlanetaryPy affiliated packages are autonomous projects created and 
+managed by their own Technical Committees (TC), not the PlanetaryPy TC.
+
+These are primarily software projects of various kinds which already
+exist separately.  They benefit from being PlanetaryPy affiliated packages
+in the following ways: 
+
+1. The PlanetaryPy TC helps them adopt or formalize governance and
+	community standards to foster their existance as an open
+	source community.
+2. They gain the ability to easily reach out to the Planetary
+	Software TC for consultation or mentorship on technical
+	or governance issues.
+3. Being recognized as a PlanetaryPy affiliated package lets others
+	know that their project conforms to community best practices
+	which are likely to result in long-term success for their
+	project.
+
+## Becoming an Affiliated Package
+
+If you are a developer of a planetary science package and would like your
+package to become affiliated with the PlanetaryPy Project, please take
+a look at the [instructions for proposing an affiliated package][review-process]. We
+recommend that you also take a look at the [guidelines for reviewing
```

Comment:
this link might be wrong? i guess the applicant wouldn't look at the review-process? Or would they?

Reply:
I agree, the name of the file is long and awkward(Affiliated-Package-Review-Process.md, thanks Astropy), but it does start with this sentence:

> This document describes the procedure for proposing, reviewing, and making a decision about PlanetaryPy-affiliated packages. 

I'm pretty sure this is the document that we want to direct them to.

Reply:
no, what i tried to remark is that "instructions for proposing" and "review process" are 2 different documents i thought?

Reply:
i see now, that `review process` is just a confusing shortcut for that document. that's fine.